### PR TITLE
Fix list of possible items in CreateRequest component

### LINKED ISSUE #301: BHoM_UI: CreateRequest exposes all adapter Create methods instead of Request only
#### Description:

As in the description, CreateRequest seems to expose all `Create` methods including the ones that do not output `IRequests`:

image

### PR DESCRIPTION
**Pull request by adecler**
### Issues addressed by this PR

Closes #301

### Test files
Just make sure that the menu of the CreateRequest component only allows to produce `IRequest`

### CODE REVIEW

**Review by pawelbaran — DISMISSED** [review]
LGTM

File: UI_Engine/Query/Items.cs [hunk 1]
```diff
@@ -107,7 +107,7 @@ public static IEnumerable<Type> ConstructableRequestItems()
         {
             return Engine.Reflection.Query.BHoMTypeList()
                 .Where(x => x != null && x.Namespace.StartsWith("BH.oM.Adapters") && !x.IsNotImplemented() && !x.IsDeprecated() && !x.IsEnum && !x.IsAbstract)
```
Comment (IsakNaslundBh) [inline UI_Engine/Query/Items.cs:109]: Do not think we should/need to filter only items in the adapter namespace. The `FitlerRequest` for example sits in `BH.oM.Data` and there are other requests available in Structures for example.

```suggestion
                .Where(x => x != null  && !x.IsNotImplemented() && !x.IsDeprecated() && !x.IsEnum && !x.IsAbstract)
```

**Review by IsakNaslundBh — CHANGES_REQUESTED** [review]
The change fixes the issue with non-IRequest showing up, but think the property assignment methods for non-adapterspecific requests are missing, see suggestion below.

File: UI_Engine/Query/Items.cs (hunk 1 above)
Comment (pawelbaran) [reply UI_Engine/Query/Items.cs:109]: Hmm, seems to work fine for me functionality wise, even though the code would say something else:

image

Comment (IsakNaslundBh) [reply UI_Engine/Query/Items.cs:109]: That is only the create methods, not the property assignment methods (automatic creates). That is at least the only ones showing up for me.

Comment (pawelbaran) [reply UI_Engine/Query/Items.cs:109]: Right. I stopped at the stage where the `Create` methods were needed, so simply added all of them. But yeah, would be great to be able to get rid of them and stick to the auto creates.

**Review by IsakNaslundBh — APPROVED** [review]
LGTM

**Pull request merged**